packages/@okta/vuepress-theme-prose/components/Home.vue

```diff
@@ -1,12 +1,23 @@
 <template>
   <div class="homepage content">
     <div class="content--container">
-       <div
-        class="sidebar-area" 
-      >
-        <Sidebar/>
-      </div> 
+      <div class="sidebar-area">
+        <Sidebar />
+      </div>
       <div class="width-adjuster">
+        <AnnouncementBanner>
+          <template v-slot:description>
+            <SmartLink :item="{ link: '/signup/' }"
+              >Developer sign ups</SmartLink
+            >
+            now utilize Okta Identity Engine beginning March 1st, 2022.
+            <SmartLink :item="{ link: '/docs/guides/oie-intro/' }"
+              >Click here</SmartLink
+            >
```

Comment:
nit: formatting is a little weird here and on the previous lines too

Reply:
I'll try to run the formatter again, so many oddities in here ^^